test/analyze_solution_test.rb

```diff
@@ -0,0 +1,20 @@
+require "test_helper"
+
+class AnalyzeSolutionTest < Minitest::Test
+
+  def test_that_it_constantizes_correctly
```

Comment:
`constantize` is a method that finds a declared constant that matches the name. [Docs here](https://apidock.com/rails/String/constantize).

Reply:
Sorry, I should have been more clear. I know what active support's constantize does, I don't understand what it has to do with analyzing the solutions.

Reply:
So the test aimed to test that the linking between "two-fer" as a string passed in from the user correctly executes the correct analyzer and checks for the correct files etc:
```
    classified_exercise = exercise_slug.tr('-', '_').classify
    results = "#{classified_exercise}::Analyze".constantize.(code_to_analyze)
```

It could definitely be better named (I think it expanded from its original intention!)  but I think it's value as a test is ok. Thoughts?

(Also, I feel like we shouldn't get bogged down on this point too much :))

Reply:
Aaaaah, I totally get it. Sorry. Yes. I think the most important thing from my perspective was just to understand why the test existed :)